plugins/tray/trayplugin.cpp

```diff
@@ -327,13 +327,15 @@ void TrayPlugin::initSNI()
 void TrayPlugin::sniItemsChanged()
 {
     const QStringList &itemServicePaths = m_sniWatcher->registeredStatusNotifierItems();
+    QDBusConnection conn = QDBusConnection::sessionBus();
```

Comment:
The "conn" is first used at the 350 line, should define it at the first use nearby.